music_server.py

```diff
@@ -315,13 +348,8 @@ def run_server():
    global curr_song
    global next_song
 
-   x_axis = np.linspace(0, 440*2*np.pi, 44100)
-   curr_song = int_array_to_bytes(2**14*np.sin(x_axis))
+   curr_song = int_array_to_bytes(np.zeros(44100))
```

Comment:
Is the 44100 intentional or is it supposed to be using a I2S sample rate like referenced in nicer_rack_esp.ino?